zenoh/tests/shm.rs

```diff
@@ -11,188 +11,194 @@
 // Contributors:
 //   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
 //
-#[cfg(all(feature = "unstable", feature = "shared-memory"))]
```

Comment:
This test shouldn't be run at all if not with `#[cfg(all(feature = "unstable", feature = "shared-memory"))]`. 
What's the reason of removing this?

Reply:
I didn't removed it. I used a inner attribute instead of a simple attribute on a module, so I removed the useless module.

Reply:
Rigth, I had missed the `!` :)